BaikalDataGenerator/Source/render.h

```diff
@@ -50,40 +50,36 @@ struct CameraInfo
     Baikal::CameraType type;
 };
 
+struct OutputDesc;
```

Comment:
(minor) I think, _OutputInfo_ is better name

Reply:
Done